usermods/smartnest/usermod_smartnest.h

```diff
@@ -168,4 +172,39 @@ class Smartnest : public Usermod
   {
     return USERMOD_ID_SMARTNEST;
   }
+
+  /**
+   * setup() is called once at startup to initialize the usermod.
+   */
+  void setup() {
+    // Initialization code here
+    if (!initialized) {
```

Comment:
Why is `initialized` needed? This function is called once (isn't it?)

Reply:
Exactly my thoughts. 😄

Reply:
Fixed up